scripts/integration/axiom/compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+  postgres:
+    image: postgres:${AXIOM_VERSION}
```

Comment:
This is a bit of a wart that I will be resolving with a follow-up PR. The variable `AXIOM_VERSION` is produced from the `version` value in the `matrix` in `test.yaml`, where `AXIOM` is created from the test name. This isn't actually the "axiom" version, it's just a version field on the axiom test. Terribly confusing. This will be changing so I can name it as the postgres version specifically.